### ISSUE TITLE
add User Enetries

### ISSUE THREAD

**Issue by R3DJOK3R1** [jdm-contrib/justdelete.me#170]
is it possible to allow user entries of course after review

**Comment by tupaschoal** [comment]
Can you elaborate on the issue and give an example? Thanks!

**Comment by R3DJOK3R1** [comment]
Can i add a website and add steps on how to delete the account from the website ?

**Comment by Nightfirecat** [comment]
Yes, you can submit a pull request to have changes added to this repository.

**Comment by tupaschoal** [comment]
I think what he wants is a form on the website (which would automatically
be converted into a pr or something similar )

**Comment by PyroSamurai** [comment]
that sounds mighty convenient but hard to maintain.

**Comment by Nightfirecat** [comment]
I'm sorry to say, but this feature is next-to-impossible to fulfill with how we're building/serving the site right now. There are [_some_ projects trying to offer something like what you're asking](https://github.com/schneidmaster/gitreports.com), but it's outside of the scope of what we're doing. I wish there was an easy way to add that functionality, but right now, there just isn't.
